src/key.cpp

```diff
@@ -105,18 +118,24 @@ AsymmetricKey RSASpec::generate() const
 
 AsymmetricKey ECCSpec::generate() const {
```

Comment:
If these curves are that special, we might want to create a separate Keyspec type? Let's discuss this with Clemens.